nx.json

```diff
@@ -137,18 +137,15 @@
   "defaultBase": "master",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33074&commitId=5a92c8d01d4fe033085d37a565d4a75cb3ccbf9b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|12| Changed|
|Avatar Converged.badgeMask.chromium.png|12| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|2824| Changed|
|Drawer.Full Overlay RTL.chromium.png|1167| Changed|
|Drawer.overlay drawer full.chromium.png|1150| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>